### DISCUSSION TITLE
Only Show Loot Gone?

### DISCUSSION THREAD

**Discussion by BoredErica** [runelite/runelite#17841] [Q&A] [answered]
Since few days ago, Only Show Loot option seems gone. I don't want to see or get notifications for other people's drops. I see there's 'Only show own items' option but it doesn't do what I want it to do.

**Comment by DapperMickie** [comment]
Is the comment "it doesn't do what I want it to do" related to: https://github.com/runelite/runelite/issues/17834 ?

**Reply by BoredErica** [reply]
I don't think it's the same exact issue. The option "only show own items" states "Only shows items that are yours or you can pick up". This seems to be an option for Irons. I'm a main. My issue is I want to make it so I don't see other people's loot, even as a main. This is because I'm camping a location with many other players around me, but running to pick their stuff up can take a very long time due to the layout of the map.

There used to be an option called 'Only show loot' which only showed items from monsters I killed. I couldn't see other players' loot or items they dropped. I'd like that behavior to continue. Now I am getting pings about loot from monsters other players killed that are too far for me to want to get them, reducing afkability with useless pings.

**Reply by BoredErica** [reply, marked as answer]
If the problematic behavior I was describing occured was a bug, it's now been fixed.

**Reply by cdfisher** [reply]
The previous behavior was restored in 622c5152ea2e96caa97106961a15ed6e39acc2d2 so that's probably why your issue went away.
